README.md

```diff
@@ -46,37 +46,20 @@ In `$HOME/.config/nixpkgs/home.nix` add
   # ...other config, other config...
 
   programs = {
-    direnv.enable = true;
-    nix-direnv.enable = true;
-  };
-
-  programs.bash.enable = true;
-  # OR
-  programs.zsh.enable = true;
-  # Or any other shell you're using.
-}
-```
-
-**Optional**: To protect your nix-shell against garbage collection
-you also need to add these options to your Nix configuration.
-
-If you are on NixOS also add the following lines to your `/etc/nixos/configuration.nix`:
+    direnv = {
+      enable = true;
+      enableBashIntegration = true; # see note on other shells below
+      nix-direnv.enable = true;
+    };
 
-```Nix
-{
-  nix.settings = {
-    keep-outputs = true;
```

Comment:
You are right. Looks like this was historically needed: https://github.com/nix-community/nix-direnv/pull/369